src/ess/reduce/nexus/workflow.py

```diff
@@ -475,6 +471,11 @@ def _add_variances(da: sc.DataArray) -> sc.DataArray:
     return out
 
 
+definitions = snx.base_definitions()
+definitions["NXdetector"] = _StrippedDetector
+definitions["NXmonitor"] = _StrippedMonitor
```

Comment:
Can you use a function or an immutable dict instead? I don't like mutable global (public) variables.

Reply:
This comment still applies.